web/src/css_variables.js

```diff
@@ -17,14 +17,20 @@ const ms = 320; // Mobile small
 // Breakpoints for middle column
 const mc = 849; // Middle column as wide as it appears after the `sm` breakpoint
 
+// Breakpoints for shifting some compose control buttons to popover
+const cb = 1390; // for when right sidebar is expanded
+const bc = 1150; // for when right sidebar is collapsed
```

Comment:
A couple of things here:

* I don't understand where the `cb` and `bc` identifiers are coming from. We've sort of painted ourselves into a corner so far with the two-letter pattern here, but I think we should be okay about breaking that pattern so this doesn't get any harder to read and reason about.
* I'm not sure where the 1150 value is coming from; the right-hand sidebar collapses below 1200, and expands at 1200 or greater--so the explanatory comments are confusing.

Reply:
The `cb` was for compose buttons, and `bc` was just me reversing that string cause I'd run out of naming ideas for 2 characters :sweat:  

Sorry I forgot to update the comment when I changed the breakpoint for Russian :grimacing: